### ISSUE TITLE
Quectel BG95 RPi CM4 Option driver rebuild: addition

### ISSUE THREAD

**Issue by xanderwasserman** [farmjenny/FarmJenny_LTE_Border_Router_HAT#1]
Hi, great guide for the Quectel BG95 linux driver for a RPi4! I have a small addition to suggest:

I followed this guide using a the following setup:
- Device: Raspberry Pi CM4
- OS: RaspberryPi OS 64-bit
- Kernel: 5.15.32-v8+
- Modem: Quectel BG95-M3

I noticed a few differences that you might want to add to your guide. I believe that these steps would be the same/similar for a Pi4 running the 64-bit OS.

In [Step 4](https://github.com/farmjenny/FarmJenny_LTE_Border_Router_HAT/wiki/Quectel-BG95-Linux-Driver#step-4-build-and-install-the-modified-kernel):
* Sub-step 2:

Raspberry Pi 4 CM4 default build configuration:
```bash
$ cd linux
$ KERNEL=kernel8
$ make bcm2711_defconfig
```

* Sub-step 3:
Where you build and install the kernel, modules, and Device Tree blobs, the top `make -j4 zImage modules dtbs` command does not work on a RPi4 running the 64-bit RaspberryPi OS (Change `zImage` to `Image`). `arm` in these commands must also be relaced by `arm64`.
Here are the updated commands:

```bash
$ make -j4 Image modules dtbs
$ make modules_install
$ cp arch/arm64/boot/dts/*.dtb /boot/
$ cp arch/arm64/boot/dts/overlays/*.dtb* /boot/overlays/
$ cp arch/arm64/boot/dts/overlays/README /boot/overlays/
$ cp arch/arm64/boot/Image /boot/$KERNEL.img
```

Hope this helps someone.

Regards,
Alexander

Reference: https://forums.raspberrypi.com/viewtopic.php?t=290354

**Comment by rcrouthamel** [comment]
Thanks Alexander.  Will add your input shortly!  Best, Rob

**Comment by rcrouthamel** [comment]
Fixed in 9306b97894a0aec9ec09203a2c39b5c55ea1b980
